routes/tasks.js

```diff
@@ -126,4 +126,35 @@ router.post('/', authenticate, authorizeOwner, createTask, tasks.addNewTask)
  */
 router.patch('/:id', authenticate, authorizeOwner, updateTask, tasks.updateTask)
 
+/**
+ * @swagger
+ * /tasks/username:
+ *   get:
+ *     summary: Use to get all the tasks of the requested user
+ *     tags:
+ *       - Tasks
+ *     security:
+ *       - bearerAuth: []
```

Comment:
is this required? we are not checking for authentication here

Reply:
Good catch @sumitd94!

Reply:
Good catch @sumitd94!